charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -440,7 +444,12 @@ spec:
           name: authenticated-emails-secret
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
       volumes:
```

Comment:
You need to move the `volumes` key outside of the templating. In the case where either `useGlobalProxyNetworking` or `trusted_ca_bundle` evaluate to false, the `volumes` key will be undefined, which will result in a templating error.